coral-trino/src/main/java/com/linkedin/coral/trino/rel2trino/Calcite2TrinoUDFConverter.java

```diff
@@ -199,6 +198,17 @@ public RexNode visitCall(RexCall call) {
 
       final String operatorName = call.getOperator().getName();
 
+      if (operatorName.equalsIgnoreCase("item") && call.getOperands().size() == 2) {
+        return super.visitCall((RexCall) rexBuilder.makeCall(TrinoElementAtFunction.INSTANCE, call.getOperands()));
+      }
+
+      if ((operatorName.equalsIgnoreCase("regexp") || operatorName.equalsIgnoreCase("rlike"))
+          && call.getOperands().size() == 2) {
+        return super.visitCall((RexCall) rexBuilder.makeCall(
```

Comment:
qq - isnt this also a standard operator transformation where based on source operator name + operand count, we apply the target operator? why cant we move it to OperatorRenameSqlCallTransformer?

Reply:
It matches the criteria of standard operator on RelNode layer. But it does not satisfy it in SqlNode layer now. As `regexp` and `rlike` is converted as `regexp` and `rlike` in some cases and is converted as `like` for example together as `not` as `not like` operator in some cases  from from `RelNode` to `SqlNode` . So we can not only check the function name and the number of operands to determine how to convert the function name. And it is left here as an Ad Hoc transformer and refactor it later.